### PR TITLE
Update readmes to match common design (Scene category) 4/5

### PR DESCRIPTION
**Pull request by JonLavi**

### CODE REVIEW

File: scene/scene-layer-selection/README.md [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
+![Image of scene layer selection](SceneLayerSelection.png)
+
+## Use case
+
+You can select features to visually distinguish them with a selection color or highlighting. This can be useful to demonstrate the physical extent or associated attributes of a feature, or to initiate another action such as centering that feature in the scene view.
 
 ## How to use the sample
 
-Click on a building in the scene layer to select it. Unselect buildings by clicking away from the buildings.
+Click a building in the scene layer to select it. Deselect buildings by tapping away from the buildings.
```
Comment (Rachael-E) [inline scene/scene-layer-selection/README.md]:
```suggestion
Click a building in the scene layer to select it. Deselect buildings by clicking away from the buildings.
```

File: scene/scene-layer-selection/README.md [hunk 2]
```diff
@@ -1,25 +1,35 @@
...
 
 ## How it works
 
-To select geoelements in a scene layer:
-
 1. Create an `ArcGISSceneLayer` passing in the URL to a scene layer service.
-2. Use `sceneView.setOnMouseClicked` to get the screen click location `Point2D`.
-3. Call `sceneView.identifyLayersAsync(sceneLayer, point2D, tolerance, false, 1)` to identify features in the scene.
-4. From the resulting `IdentifyLayerResult`, get the list of identified `GeoElement`s with `result.getElements()`.
+2. Use `sceneView.setOnTouchListener(...)` to get the screen tap location `screenPoint`.
```
Comment (Rachael-E) [inline scene/scene-layer-selection/README.md]: reverting this back to original for Java platform ;)
```suggestion
2. Use `sceneView.setOnMouseClicked` to get the screen click location `Point2D`.
```

File: scene/scene-layer-selection/README.md [hunk 3]
```diff
@@ -1,25 +1,35 @@
...
+3. Call `sceneView.identifyLayersAsync(sceneLayer, screenPoint, tolerance, false, 1)` to identify features in the scene.
```
Comment (Rachael-E) [inline scene/scene-layer-selection/README.md]:
```suggestion
3. Call `sceneView.identifyLayersAsync(sceneLayer, point2D, tolerance, false, 1)` to identify features in the scene.
```

File: scene/scene-layer-selection/README.md [hunk 4]
```diff
@@ -1,25 +1,35 @@
...
+4. From the resulting `IdentifyLayerResult`, get the list of identified `GeoElements` with `result.getElements()`.
```
Comment (Rachael-E) [inline scene/scene-layer-selection/README.md]:
```suggestion
4. From the resulting `IdentifyLayerResult`, get the list of identified `GeoElement`s with `result.getElements()`.
```

**Review by Rachael-E — APPROVED** [review]
just some minor reverts needed here for java specific doc: approving since easy changes.

File: scene/scene-layer-selection/README.md (hunk 2 above)
Comment (JonLavi) [reply scene/scene-layer-selection/README.md]: Ohh I clearly lost focus on these ones! Sorry!!

File: scene/scene-layer-selection/README.md (hunk 3 above)
Comment (JonLavi) [reply scene/scene-layer-selection/README.md]: Good catch, this must have been wrong for a while!

**Pull request merged**